db/db_readonly_with_timestamp_test.cc

```diff
@@ -320,6 +320,312 @@ TEST_F(DBReadOnlyTestWithTimestamp,
 
   Close();
 }
+
+TEST_F(DBReadOnlyTestWithTimestamp, CompactedDBGetReadTimestampSizeMismatch) {
```

Comment:
In all these unit tests, I think we need to make sure that `CompactedDBImpl` is actually used. To do so, we can
- only one L0
- Only Lmax has data.

Reply:
Good point, I added a "CheckDBOpenedAsCompactedDB" check for these test cases using this CompactedDB criteria: https://github.com/facebook/rocksdb/blob/main/db/db_impl/compacted_db_impl.cc#L138~L144

Reply:
How about these: https://github.com/facebook/rocksdb/blob/main/db/db_impl/compacted_db_impl.cc#L146:L149?

Reply:
Added test cases for Get and MultiGet with a `CheckDBOpenedAsCompactedDBWithOnlyLmaxFiles` check.